sql/core/src/test/scala/org/apache/spark/sql/execution/SortSuite.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution
+
+import org.apache.spark.sql.catalyst.expressions.{BoundReference, Ascending, SortOrder}
+import org.apache.spark.sql.types.{IntegerType, StringType}
+
+class SortSuite extends SparkPlanTest {
+
+  test("basic sorting using ExternalSort") {
+
+    val input = Seq(
+      ("Hello", 4),
+      ("Hello", 1),
+      ("World", 8)
+    )
+
+    val sortOrder = Seq(
+      SortOrder(BoundReference(0, StringType, nullable = false), Ascending),
```

Comment:
It's a little annoying to have to manually bind these references.  It would be nice if there was some sort of rewrite that I could use that would bind references like `'_1` to the proper columns.